build.sbt

```
@@ -67,9 +67,11 @@ lazy val core = project
     libraryDependencies ++= List(
       scalameta,
       googleDiff,
-      metaconfig,
       collectionCompat
-    )
+    ) :+ (CrossVersion.partialVersion(scalaVersion.value) match {
+      case Some((2, 11)) => metaconfigFor211
+      case _ => metaconfig
+    })
```

you can't use `isScala211` here? if that's not possible, you can always go for 
```suggestion
    ),
    libraryDependencies += (if (isScala211.value) metaconfigDocFor211
                            else metaconfigDoc)
```